packages/components/src/unit-control/types.ts

```diff
@@ -75,7 +75,11 @@ export type UnitSelectControlProps = Pick< InputControlProps, 'size' > & {
 export type UnitControlProps = Omit< UnitSelectControlProps, 'unit' > &
 	Pick<
 		InputControlProps,
-		'hideLabelFromVision' | 'prefix' | '__next36pxDefaultSize'
+		| 'dragDirection'
+		| 'hideLabelFromVision'
+		| 'labelPosition'
+		| 'prefix'
+		| '__next36pxDefaultSize'
```

Comment:
UnitControl was missing some props.

Reply:
We can probably wait for the `NumberControl` refactor to TypeScript to land first, and then rebase?

Reply:
Good call 👍